airbyte-ci/connectors/pipelines/pipelines/actions/environments.py

```diff
@@ -909,19 +799,6 @@ async def with_airbyte_java_connector(context: ConnectorContext, connector_java_
     return await finalize_build(context, connector_container)
 
 
-async def get_cdk_version_from_python_connector(python_connector: Container) -> Optional[str]:
-    pip_freeze_stdout = await python_connector.with_entrypoint("pip").with_exec(["freeze"]).stdout()
-    cdk_dependency_line = next((line for line in pip_freeze_stdout.split("\n") if "airbyte-cdk" in line), None)
-    if not cdk_dependency_line:
-        return None
-
-    if "file://" in cdk_dependency_line:
-        return "LOCAL"
-
-    _, cdk_version = cdk_dependency_line.split("==")
-    return cdk_version
```

Comment:
My main question - where do we specify local cdk now? It makes sense to me that we're removing injecting it during test since we probably want to inject it in build now. However taking a look at the build PR, the only reference I see to it there is deleting "use the local cdk" code: https://github.com/airbytehq/airbyte/pull/30456/files#diff-b1c73385b3a55c0d61d11e4323076b841402c0401100fc95d902b9c4d87609feL939-L948

Reply:
@erohmensing the "local cdk" feature is not lost. It's wrapped inside an "override" function [here](https://github.com/airbytehq/airbyte/pull/30456/files#diff-d208478cc2467e3ae31f756ca741bc86e2eb1dd98ed1fffd11627fc9e819999eR35) (on the other PR)
It indeed only needs to happen at build time now.

Reply:
Aha - exactly what I was looking for. thanks!